src/benchmarklib/tpcc/tpcc_table_generator.cpp

```diff
@@ -22,9 +21,11 @@
 
 namespace opossum {
 
-TpccTableGenerator::TpccTableGenerator(const ChunkOffset chunk_size, const size_t warehouse_size)
+TpccTableGenerator::TpccTableGenerator(const ChunkOffset chunk_size, const size_t warehouse_size,
+                                       std::optional<EncodingConfig> encoding_config)
```

Comment:
Why not just use EncodingType::Unencoded instead of optional for unencoded tables?

Reply:
for simplicity. otherwise you would have to create this "unencoded" encoding each time. but i think this isn't a bad idea. i'll add `EncodingConfig::unencoded` which can then be passed around. then there is no optional and it's not super weird to create it if needed.